### ISSUE TITLE
Customizable App Name

### ISSUE THREAD

**Issue by Dualkem** [topjohnwu/Magisk#1859]
Hello! I had a problem for a while with a Romanian Internet Banking App called eToken. At first it seemed that it was able to detect root so well that even Magisk+MagiskHide+"Hide MagiskManager" was unable to hide from it, and it was pretty annoying. After experimenting with it I found out that it wasn't checking for root or even bootloader state. It was searching for "Magisk Manager" in the list of app names. Before the experiment, the eToken app ran fine. In order to test this theory I've created a mock app called "Magisk Manager"(no magisk reference in package name), without any root code in it (without any code, it was a blank screen). And, surprise : root-access-related error.

TL;DR : Alongside package name, some apps seem to also check for the app name. Maybe an option to input what name should be displayed instead of "Magisk Manager" will do the job just fine.
Also, although I don't think that this is necessarily an issue about MagiskHide, I apologize in advance if I broke some rules. Thank you!

**Comment by Didgeridoohan** [comment]
When you hide the Manager (repackage with a random name) it also changes the name to just "Manager". You likely have some other issue...

I tested the app and had no issues hiding Magisk from it

**Comment by Dualkem** [comment]
> 
> 
> When you hide the Manager (repackage with a random name) it also changes the name to just "Manager". You likely have some other issue...
> 
> I tested the app and had no issues hiding Magisk from it

Thank you! It seems like I forgot to delete the mock app from Island, and so the eToken app managed to read it's name. Should I keep this issue as a proposal or should I close it?

**Comment by Didgeridoohan** [comment]
I would say it can be closed. User customisable app (and even package) name has been discussed before and it's generally been considered to not be worth the effort.
